CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 
 ##### Enhancements
 
+* Added `ensure_bundler!` flag to the Podfile DSL.  
+  [Sean Reinhardt](https://github.com/seanreinhardtapps)
+  [#7432](https://github.com/CocoaPods/CocoaPods/issues/9325)
```

Comment:
@seanreinhardtapps #7432 does not match the issue 9325 in this line.

Reply:
:man_facepalming: